it/stubs/FinancialDataStub.scala

```diff
@@ -95,6 +122,18 @@ object FinancialDataStub extends WireMockMethods {
        |  }""".stripMargin
   )
 
+  val validChargesJson: JsValue = outstandingTransactions(
+    "\"VAT Return Debit Charge\"",
+    "\"VAT Return Credit Charge\"",
+    "\"VAT OA Debit Charge\""
+  )
+
+  val invalidChargesJson: JsValue = outstandingTransactions(
```

Comment:
A bit misleading - one of these is a valid charge type. Could you instead remove the valid one and then your test can confirm the empty sequence?